components/config/prefs.rs

```diff
@@ -225,6 +225,9 @@ mod gen {
                         enabled: bool,
                     }
                 },
+                abort_controller: {
+                    enabled: bool,
```

Comment:
We should add a doc comment about what this enables.

Reply:
Done.